src/Adapters/AdapterLoader.js

```diff
@@ -53,13 +53,15 @@ export async function loadModule(modulePath) {
   } catch (err) {
     if (err.code === 'ERR_REQUIRE_ESM') {
       module = await import(modulePath);
-      if (module.default) {
-        module = module.default;
-      }
     } else {
       throw err;
     }
   }
+
+  if (module.default) {
```

Comment:
It seems that this change is different from the change in https://github.com/parse-community/parse-server/pull/9524; deliberately?